jobs/periodic/revalidate-streaks/index.ts

```diff
@@ -0,0 +1,35 @@
+import { checkUserAchievement } from '../../../common/achievement';
+import { UserAchievementTemplate } from '../../../common/achievement/types';
+import { prisma } from '../../../common/prisma';
+
+export enum StreakTimeFrame {
+    WEEK = 'by_weeks',
+    MONTH = 'by_months',
+}
+
+export default async function revalidateStreaks(timeFrame: StreakTimeFrame) {
```

Comment:
can you please add a comment why we need a cronjob for streaks? (if possible 😄 )

Reply:
Done!